packages/directlinespeech/src/createAdapters.js

```diff
@@ -0,0 +1,161 @@
+    window.navigator.language) ||
+    'en-US',
+  speechSynthesisDeploymentId,
+  speechSynthesisOutputFormat,
+  textNormalization,
+  userID,
+  username
+}) {
+  if (!fetchCredentials) {
+    throw new Error('"fetchCredentials" must be specified.');
+  }
+
+  const { authorizationToken, region, subscriptionKey } = await resolveFunctionOrReturnValue(fetchCredentials);
+
+  if (
+    (!authorizationToken && !subscriptionKey) ||
+    (authorizationToken && subscriptionKey) ||
+    (authorizationToken && typeof authorizationToken !== 'string') ||
+    (subscriptionKey && typeof subscriptionKey !== 'string')
+  ) {
+    throw new Error(
+      '"fetchCredentials" must return either "authorizationToken" or "subscriptionKey" as a non-empty string only.'
+    );
+  }
+
+  if (typeof enableTelemetry !== 'undefined') {
+    console.warn(
+      'botframework-directlinespeech: Telemetry options is not supported yet. Please refer to Cognitive Services documentation for details.'
+    );
+  }
+
+  if (!region || typeof region !== 'string') {
+    throw new Error('"fetchCredentials" must return "region" as a non-empty string.');
+  }
+
+  if (speechRecognitionEndpointId) {
+    console.warn(
+      'botframework-directlinespeech: Custom Speech is currently not supported, ignoring speechRecognitionEndpointId.'
+    );
+  }
+
+  if (speechSynthesisDeploymentId) {
+    console.warn(
+      'botframework-directlinespeech: Custom Voice is currently not supported, ignoring speechSynthesisDeploymentId.'
+    );
+  }
+
+  if (speechSynthesisOutputFormat) {
+    console.warn(
+      'botframework-directlinespeech: Synthesis output format is currently not supported, ignoring speechSynthesisOutputFormat.'
+    );
+  }
+
+  if (textNormalization) {
+    console.warn(
+      'botframework-directlinespeech: Text normalization is currently not supported, ignoreing "textNormalization".'
+    );
+  }
+
+  if (userID || username) {
+    console.warn(
+      'botframework-directlinespeech: Custom "userId" and "username" are currently not supported, ignoring.'
+    );
+  }
+
+  let config;
+
+  if (authorizationToken) {
+    config = BotFrameworkConfig.fromAuthorizationToken(authorizationToken, region);
+  } else {
+    config = BotFrameworkConfig.fromSubscription(subscriptionKey, region);
+  }
+
+  // Supported options can be found in DialogConnectorFactory.js.
+
+  config.setProperty(PropertyId.SpeechServiceConnection_RecoLanguage, speechRecognitionLanguage);
+
+  // None of the following works for setting output format.
+
+  // config.setProperty(PropertyId.SpeechServiceResponse_OutputFormatOption, OutputFormat[OutputFormat.Detailed]);
+  // config.setProperty(PropertyId.SpeechServiceResponse_RequestDetailedResultTrueFalse, true);
+  // config.setProperty(OutputFormatPropertyName, OutputFormat[OutputFormat.Detailed]);
+  // config.setServiceProperty(PropertyId.SpeechServiceResponse_RequestDetailedResultTrueFalse, "true", ServicePropertyChannel.UriQueryParameter);
```

Comment:
delete? or expand on the comment on line 100

Reply:
Expanded.